run.py

```diff
@@ -20,6 +20,8 @@ def run(command, env={}):
     if process.returncode != 0:
         raise Exception("Non zero return code: %d"%process.returncode)
 
+__version__ = open('/version').read()
```

Comment:
I think it is better to explicitly close the file handle, why not using a context?